sdk/keyvault/keyvault-certificates/CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Release History
 
+## 4.0.0-preview.9 (2019-11-04)
+
+- Fixed parameter names and access modifiers to be on par of other language implementations.
+- Updated dependencies.
+- Cleaned up the exported types.
+- Renamed `Certificate` to `KeyVaultCertificate`.
+- Ensured that all the optional parameters have this format `{PascalCaseMethodName}Options`.
+- Date properties now end in `On`.
+- Anything `.*CertificateContacts.*` was changed to be `.*Contacts.*` except for the main class `CertificateContacts`.
+- Anything `.*CertificateIssuers.*` was changed to be `.*Issuers.*` except for the main class `CertificateIssuers`.
+- `restoreCertificate` is now `restoreCertificateBackup`.
+- `AdministratorDetails` is now `AdministratorContact`.
+- `getCertificateWithPolicy` was renamed `GetCertificate` and `GetCertificate` was renamed `GetCertificateVersion`.
```

Comment:
capitalization looks wrong. should be `getCertificate` and `getCertificate` was renamed to `getCertificateVersion`, I believe